pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows/guide.fr-fr.md

```diff
@@ -0,0 +1,98 @@
+
+Recherchez « Boot » dans la zone **Informations générales** et cliquez sur `...`{.action} puis sur `Modifier`{.action}.
+
+![Changer le mode de démarrage](images/rescue-mode-001.png){.thumbnail}
+
+Dans la page suivante, sélectionnez **Booter en mode rescue**. Choisissez  `rescue-customer-windows`{.action} (voir la [section du guide ci-dessous](#windowsrescue). Spécifiez une autre adresse de messagerie si vous ne souhaitez **pas** que les identifiants de connexion soient envoyées à l'adresse principale de votre compte OVHcloud.
+<br>Cliquez sur `Suivant`{.action} et `Valider`{.action}.
+
+![Mode rescue-customer](images/manager-rescue-windows-menu.png){.thumbnail}
+
+Une fois la modification terminée, cliquez sur `...`{.action} à droite de « Statut » dans la zone intitulée **État des services**. 
+<br>Cliquez sur `Redémarrer`{.action} et le serveur redémarrera en mode rescue. Cette opération peut prendre quelques minutes. 
+<br>Vous pouvez vérifier l'avancement sous l'onglet `Tâches`{.action}. Une fois le système de rescue disponible, un e-mail vous sera envoyé, contenant les identifiants (dont le mot de passe de connexion)  de l'utilisateur « Administrator » du mode rescue.
+
+![Redémarrer le serveur](images/rescue-mode-02.png){.thumbnail}
+
+Lorsque vous aurez terminé vos tâches en mode rescue, n'oubliez pas de redéfinir le mode de boot (netboot) sur `Booter sur le disque dur`{.action} puis redémarrez votre serveur.
+
+
+### Connexion au _rescue-customer-windows_
+
+Une fois le mot de passe récupéré, vous aurez trois possibilité pour vous connecter au serveur:
+- Remote Desktop Protocol (RDP)
+- SSH (Composant officiel Windows OpenSSH Server)
+- KVM IP (Si votre serveur le permet) 
+
+> [!warning]
+>
+> Dans tous les cas, le mot de passe vous sera demandé.  
+> L'utilisateur permettant de se connecter est `Administator`.  
+> Le mot de passe est transmis via un lien `secret-as-a-service`.
+
+#### Utilisation du KVM IP
+
+Il est possible, à partir de l'écran de login du KVM,
+![KVM Login Screen](images/rescue-kvm-login-screen.png){.thumbnail}
+de sélectionner une autre langue que celle par défaut.
+![KVM Language Screen](images/rescue-kvm-login-language.png){.thumbnail}
+
+
+Il est possible de changer les options d'accessibilité et d'activer le clavier virtuel:
+![KVM accessibilty Screen](images/rescue-kvm-login-accessibility.png){.thumbnail}
+![KVM keyboard Screen](images/rescue-kvm-login-keyboard.png){.thumbnail}
+
+
+### Monter les disques
+
+Il est possible que les disques attachés soient vu comme des `Volumes Dynamiques`. Pour pouvoir les utiliser, veuillez vous référer à la [documentation officielle Microsoft]( https://learn.microsoft.com/en-us/troubleshoot/windows-server/backup-and-storage/troubleshoot-disk-management#a-dynamic-disks-status-is-foreign)
+
+## Aller plus loin
+
```

Comment:
Liens:
- Vérifier les valeurs SMART des disques, télécharer et installer CrystalDiskInfo : https://crystalmark.info/redirect.php?product=CrystalDiskInfoInstaller
- Accéder à ses disques dynamiques (soft raid) : https://learn.microsoft.com/troubleshoot/windows-server/backup-and-storage/troubleshoot-disk-management#a-dynamic-disks-status-is-foreign
- Réinitialiser le mot de passe de son serveur : /pages/bare_metal_cloud/dedicated_servers/changing-admin-password-on-windows

Reply:
la réinit du mdp est déjà notée plus haut (ligne 13)

smartctl? https://www.smartmontools.org/wiki/Download